components/Auction/Auction.module.css

```diff
@@ -1,167 +1,158 @@
-.newAuctionBtnContainer {
-  text-align: center;
+.auction {
+  display: flex;
+  flex-direction: column;
+  padding: 1.5rem;
+  gap: 1rem;
 }
 
-.newAuctionBtn {
-  font-weight: 550;
-  height: 50px;
-  font-size: 1.2rem;
-  width: 14rem;
-  border: none;
+.auctionButtonContainer {
   text-align: center;
-  padding: 2px 10px;
-  border-radius: 10px;
-  background-color: #101f92;
-  color: white;
+  margin-left: auto;
 }
 
-.mainContainer {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  margin-bottom: 100px;
+.auctionButton {
+  max-width: 13.125rem;
+  width: 100%;
+  height: 3.125rem;
+  padding: 2px 0.625rem;
```

Comment:
why `px` and why it's combined with rem

Reply:
If values are less than 8 then I am not using `rem` as unit for them.

Reply:
Using px as unit for values less than 8

Reply:
just want to know what's the benefit of doing like this, imo for padding it's better to use `rem`

Reply:
This is not about benefit, but more of personal prefference.